pkg/ipmi/ipmi.go

```diff
@@ -310,3 +331,69 @@ func (i *IPMI) SetSystemFWVersion(version string) error {
 
 	return nil
 }
+
+func (i *IPMI) GetDeviceID() (*ipmDevIdRsp, error) {
```

Comment:
I'm sorry, one last thing: exported functions should return exported types, otherwise users of the function cannot access the information. s/ipmiDevIdRsp/DevID

(remove ipmi, because externally, it'd be "ipmi.IPMIDevID" and that repetition is unnecessary, and "Rsp" doesn't seem necessary either)

Reply:
Updated for exported types.